analyzer/codechecker_analyzer/cmd/parse.py

```diff
@@ -19,7 +19,7 @@
 from operator import itemgetter
 import sys
 import traceback
-from typing import List, Dict, Tuple, Set, Union
+from typing import List, Dict, Tuple, Set, Union, Callable
```

Comment:
It would be good to write these imports in alphabetical order:
```suggestion
from typing import Callable, Dict, List, Set, Tuple, Union
```